homeassistant/components/homeassistant_hardware/const.py

```diff
@@ -4,5 +4,18 @@
 
 LOGGER = logging.getLogger(__package__)
 
+DOMAIN = "homeassistant_sky_connect"
```

Comment:
It is probably not relevant as this is no complete/real domain, but should this be `homeassistant_hardware`? :thinking: 
```suggestion
DOMAIN = "homeassistant_hardware"
```